Frontend/Pages/Main/Start.razor

```diff
@@ -80,12 +98,28 @@
 
     private string? _bookmarkStateFilter = "bi-bookmark";
 
+    private int numberOfPages;
+
+    private int numberOfEntries;
+
+
+
+    /* is needed to allow different number of entries per page
```

Comment:
is this still required?

Reply:
it could be an additional feature one could implement one dayc
its not needed but if one wants to add the feature it would be helpful
should i delete it?

Reply:
No then its fine.